common/djangoapps/third_party_auth/models.py

```diff
@@ -378,7 +378,8 @@ class SAMLProviderConfig(ProviderConfig):
         help_text="URN of SAML attribute containing the user's email address[es]. Leave blank for default.")
     automatic_refresh_enabled = models.BooleanField(
         default=True, verbose_name="Enable automatic metadata refresh",
-        help_text="When checked, the SAML provider's metadata will be included in the automatic refresh job, if configured.")
+        help_text="When checked, the SAML provider's metadata will be included in the automatic refresh job, "
+                  "if configured.")
```

Comment:
Sure! I have a nit-picky nature so anything to refactor in the `third_party_auth` app got my attention. But I understand that the diffs can get messy.